beacon-chain/blockchain/process_block_helpers.go

```diff
@@ -129,6 +128,94 @@ func (s *Service) sendLightClientFeeds(cfg *postBlockProcessConfig) {
 	}
 }
 
+// saveLightClientUpdate saves the light client update for this block
+// if it's better than the already saved one, when feature flag is enabled.
+func (s *Service) saveLightClientUpdate(cfg *postBlockProcessConfig) {
```

Comment:
General comment for all error logs in this function: Can you add something like "saving light client update failed" to each log? Because otherwise it will be hard to understand where the error comes from.